src/studio/src/designer/backend/Services/Implementation/SourceControlSI.cs

```diff
@@ -220,8 +220,9 @@ public void PushChangesForRepository(CommitInfo commitInfo)
         /// </summary>
         /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
         /// <param name="repository">The name of the repository</param>
-        public async Task Push(string org, string repository)
+        public async Task<bool> Push(string org, string repository)
         {
+            bool pushSuccess = true;
             string localServiceRepoFolder = _settings.GetServicePath(org, repository, AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext));
             var watch = System.Diagnostics.Stopwatch.StartNew();
```

Comment:
would this be the right time to remove the watch used fore debugging way back when??

Reply:
Agreed, fixed.